frontend/src/Organize/OrganizePreviewModalContent.js

```diff
@@ -107,13 +106,7 @@ class OrganizePreviewModalContent extends Component {
 
           {
             !isFetching && isPopulated && !items.length &&
-              <div>
-                {
-                  renameTracks ?
-                    <div>Success! My work is done, no files to rename.</div> :
-                    <div>Renaming is disabled, nothing to rename</div>
-                }
-              </div>
+              <div>Success! My work is done, no files to rename.</div>
```

Comment:
Don't know that we should do the same thing for renaming as for tagging. Renaming has a single checkbox that hides naming settings if turned off. Thus user is going to be renaming something that they have no control of the format. If we do this we should likely always show the track format naming field in settings whether renaming is enabled or not.